### PR TITLE
Blog for Android 2023.3

### PR DESCRIPTION
**Pull request by jpelgrom**
## Proposed change

Adds a blog post for the upcoming Android 2023.3 release. Specific date TBD but should be this month.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-03-25-android-20233.markdown [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ - Release-Notes
+og_image: /images/blog/2023-03-25-android/social.png
+---
+
+![Home Assistant logo with 'Android 2023.3' text](/images/blog/2023-03-25-android/social.png)
+
+It's finally here, one of the most common feature requests for the Home Assistant
+Android Companion app: you can now add multiple servers to your app, send sensor
+updates to all connected servers and easily mix and match widgets, tiles and
+controls. 🎉
```
Comment (balloob) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
It's finally here, one of the most common feature requests for the Home Assistant
Android Companion app: you can now configure multiple Home Assistant servers. The app can send sensor
updates to all servers and easily mix and match widgets, tiles and
controls for different servers 🎉
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Plus, a preview of the app is now available on the Amazon Appstore on Windows 11
+devices, which allows you to quickly connect your computer to Home Assistant.
```
Comment (balloob) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
Plus, a preview of the Home Assistant app is now available on the Amazon Appstore on Windows 11, which allows you to quickly connect your computer to Home Assistant.
```

**Comment by jpelgrom** [comment]
@frenck Could you help with a header/social image in the 'usual style' for the core releases? It appears you have the template. The app doesn't really have integrations to show, but I was thinking maybe a Windows logo and servers sheet could work.
[assets.zip](https://github.com/home-assistant/home-assistant.io/files/11063312/assets.zip)

File: source/_posts/2023-03-25-android-20233.markdown [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+categories:
+ - Release-Notes
+og_image: /images/blog/2023-03-25-android/social.png
+---
+
+![Home Assistant logo with 'Android 2023.3' text](/images/blog/2023-03-25-android/social.png)
+
+It's finally here, one of the most common feature requests for the Home Assistant
+Android Companion app: you can now configure multiple Home Assistant servers. The
+app can send sensor updates to all servers and easily mix and match widgets, tiles
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
app can send sensor updates to all servers and easily mix and match widgets, tiles,
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+and controls for different servers 🎉
+
+Plus, a preview of the Home Assistant app is now available on the Amazon Appstore
+on Windows 11, which allows you to quickly connect your computer to Home Assistant.
+
+There are also other changes that add some polish to the app and improve support
+for features in the latest Home Assistant releases, so let's dig in to the update!
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
for features in the latest Home Assistant releases, so let's dig into the update!
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+## Configure multiple Home Assistant servers
+
+Sometimes, one Home Assistant instance is not enough. You like Home Assistant so
+much that you decide to help someone else get started, there's a second home in
+your life, or you regularly visit someplace else where they're using Home Assistant.
+
+Before this update, if you wanted to access more than one server it was a bit of a
+hassle. Switching apps, creating bookmarks, using different profiles. Let alone
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
hassle. Switching apps, creating bookmarks, and using different profiles. Let alone
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+connecting your Android device to multiple servers for features like sensors
+and notifications.
+
+This issue now belongs to the past, as there's a new option in [Settings] >
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
This issue now belongs to the past, as there's a new option in **{% my config title="Settings" %}** >
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+Companion app to configure  a second server, and another one after that, and another...
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
**Companion app** to configure a second server, and another one after that, and another...
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+They will all show up in the app, and you can use all features you've come to
+expect like lots of sensors for your device and support for push notifications.
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
expect, like lots of sensors for your device and support for push notifications.
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+Each one gets individual settings, so if you don't want to share your step count
+and enable location tracking everywhere, that's OK.
+
+![Screenshots of a phone showing different dashboards in the app, one of them showing a pop-up titled 'Choose server'](/images/blog/2023-03-25-android/multiserver.png)
+
+That's not all - you can use all servers _outside_ the app as well, mixing and
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
That's not all - you can also use all servers _outside_ the app, mixing and
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+Toggling lights from three different homes side-by-side? No problem!
+
+Note: the Wear OS app does not yet support multiple servers.
+
+[Settings]: https://my.home-assistant.io/redirect/config/
+
+## Now available on Windows 11 (preview)
+
+The Home Assistant Android Companion app is now available on [supported Windows 11 devices]
+via the Amazon Appstore. While there are already great options to integrate your
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
The Home Assistant Android Companion app is now available via the Amazon Appstore
on [supported Windows 11 devices]. While there are already great options to integrate your
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+computer out there from the community, the Companion app makes it easy to start
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
computer from the community out there, the Companion app makes it easy to start
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+
+![Screenshot of the Home Assistant Android app on a Windows 11 device](/images/blog/2023-03-25-android/windows.png)
+
+Windows does not always provide correct information to the app and may shut down
+background work, including sensors, after several minutes, which is why we are
+releasing Windows 11 support for the app as a preview. Android support in Windows
+11 continues to get better and we intend to make adjustments to improve the
+experience in the future.
+
+TODO: link to Microsoft Store
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]: TODO

File: source/_posts/2023-03-25-android-20233.markdown [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+[supported Windows 11 devices]: https://support.microsoft.com/windows/f8d0abb5-44ad-47d8-b9fb-ad6b1459ff6c
+
+## Refining Wear OS
+
+The Home Assistant app has been available for a bit over a year now on Wear OS
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
The Home Assistant app has been available for over a year now on Wear OS
```

File: source/_posts/2023-03-25-android-20233.markdown [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+watches, and continues to improve. Earlier this year initial support for Assist was
```
Comment (frenck) [inline source/_posts/2023-03-25-android-20233.markdown]:
```suggestion
watches and continues to improve. Earlier this year, initial support for Assist was
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
> @frenck Could you help with a header/social image in the 'usual style' for the core releases?

Sure! Here you go:

Home Assistant - Blog - Android 2023 3

If you want a different one, or need changes, feel free to reach out (here or on Discord).

**Comment by jpelgrom** [comment]
Thanks for the comments. Image looks better than expected, no changes needed!

File: source/_posts/2023-03-25-android-20233.markdown (hunk 12 above)
Comment (jpelgrom) [reply source/_posts/2023-03-25-android-20233.markdown]: Removed as it doesn't look like deeplinks are available

**Comment by dshokouhi** [comment]
Update is now live in the play store :)

**Review by dshokouhi — DISMISSED** [review]
Looks good to me! Great stuff this release!!!!

**Review by balloob — APPROVED** [review]

**Pull request merged**